src/libraries/System.Security.AccessControl/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This should be a `.LibraryBuild` file.

See https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/ILLink-files.md for more info.

Also, see:
https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/eng/illink.targets#L28-L29

Reply:
oops, good catch, sorry I totally forgot about this.